foundation-ui/auth/AuthService.js

```diff
@@ -0,0 +1,67 @@
+import {EventEmitter} from 'events';
+
+import Authorizer from './Authorizer';
+import {CHANGE} from './AuthEvent';
+
+/**
+ * List of authorizers
+ *
+ * @type {Array.<Authorizer>}
+ */
+const authorizers = [];
+
+/**
+ * AuthService
+ */
+class AuthService extends EventEmitter {
+
+  /**
+   * Register authorizer
+   *
+   * @param {Authorizer} authorizer
+   */
+  registerAuthorizer(authorizer) {
+    if (!authorizer || !(authorizer instanceof Authorizer)) {
+      if (global.__DEV__) {
+        throw new TypeError('Provided component must be a ' +
+            'React.Component constructor or a stateless functional component.');
+      }
+
+      return;
+    }
+
+    authorizers.push(authorizer);
+    this.emit(CHANGE);
+  }
+
+  /**
+   * Unregisters authorizer
+   *
+   * @param  {Authorizer} authorizer
+   */
+  unregisterAuthorizer(authorizer) {
+    let i = authorizers.length;
+    while (--i >= 0) {
+      if (authorizers[i] === authorizer) {
+        authorizers.splice(i, 1);
+        break;
```

Comment:
Good point, I'll make sure that it is not possible to register an authorizer multiple times.